DQM/SiStripMonitorClient/test/mergeBadChannel_Template_cfg.py

```diff
@@ -0,0 +1,92 @@
+import FWCore.ParameterSet.Config as cms
+
+import FWCore.ParameterSet.VarParsing as VarParsing
+
+process = cms.Process("BadChannelMerge")
+
+#prepare options
+
+options = VarParsing.VarParsing("analysis")
+
+options.register ('globalTag',
+                                    "DONOTEXIST::All",
+                                    VarParsing.VarParsing.multiplicity.singleton, # singleton or list
+                                    VarParsing.VarParsing.varType.string,          # string, int, or float
+                                    "GlobalTag")
+options.register ('dqmFile',
+                                    "",
+                                    VarParsing.VarParsing.multiplicity.singleton, # singleton or list
+                                    VarParsing.VarParsing.varType.string,          # string, int, or float
+                                    "DQM root file")
+options.register ('runNumber',
+                                    1,
+                                    VarParsing.VarParsing.multiplicity.singleton, # singleton or list
+                                    VarParsing.VarParsing.varType.int,          # string, int, or float
+                                    "run number")
+options.parseArguments()
+
+
+process.MessageLogger = cms.Service("MessageLogger",
+    destinations = cms.untracked.vstring('cout','cerr','MergedBadComponents'), #Reader, cout
+    categories = cms.untracked.vstring('SiStripQualityStatistics'),
+
+    debugModules = cms.untracked.vstring('siStripDigis', 
+                                         'siStripClusters', 
+                                         'siStripZeroSuppression', 
+                                         'SiStripClusterizer',
+                                         'siStripOfflineAnalyser'),
+    cerr = cms.untracked.PSet(threshold = cms.untracked.string('ERROR')
+                              ),
+    cout = cms.untracked.PSet(threshold = cms.untracked.string('INFO'),
+                                default = cms.untracked.PSet(limit=cms.untracked.int32(0))
+                              ),
+    MergedBadComponents = cms.untracked.PSet(threshold = cms.untracked.string('INFO'),
+                                default = cms.untracked.PSet(limit=cms.untracked.int32(0)),
+                                SiStripQualityStatistics = cms.untracked.PSet(limit=cms.untracked.int32(100000))
+                                )
+                                    
+)
+process.load('Configuration.Geometry.GeometryIdeal_cff')
+process.load("Configuration.Geometry.GeometryReco_cff")
```

Comment:
Done

Il 18/08/2016 17.11, boudoul ha scritto:
>
> In DQM/SiStripMonitorClient/test/mergeBadChannel_Template_cfg.py 
> <https://github.com/cms-sw/cmssw/pull/15517#discussion_r75327092>:
>
> > +                                         'siStripZeroSuppression',
> > +                                         'SiStripClusterizer',
> > +                                         'siStripOfflineAnalyser'),
> > +    cerr = cms.untracked.PSet(threshold = cms.untracked.string('ERROR')
> > +                              ),
> > +    cout = cms.untracked.PSet(threshold = cms.untracked.string('INFO'),
> > +                                default = cms.untracked.PSet(limit=cms.untracked.int32(0))
> > +                              ),
> > +    MergedBadComponents = cms.untracked.PSet(threshold = cms.untracked.string('INFO'),
> > +                                default = cms.untracked.PSet(limit=cms.untracked.int32(0)),
> > +                                SiStripQualityStatistics = cms.untracked.PSet(limit=cms.untracked.int32(100000))
> > +                                )
> > +
> > +)
> > +process.load('Configuration.Geometry.GeometryIdeal_cff')
> > +process.load("Configuration.Geometry.GeometryReco_cff")
>
> please replace those 2 lines by
> process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/cms-sw/cmssw/pull/15517/files/5ed6cff3e92955653a617a0a42d6351e2a70f03a#r75327092>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AKYUWk-lWaJ6v2eOB0IExPztRGJ6vX1fks5qhHYhgaJpZM4Jnlqx>.
>